pulsar-client-cpp/include/pulsar/MessageId.h

```diff
@@ -22,13 +22,12 @@
 #include <iosfwd>
 #include <stdint.h>
 #include <boost/shared_ptr.hpp>
+#include <lib/MessageIdImpl.h>
```

Comment:
We cannot include `lib/*.h` headers from public API. We need to just do the forward declaration